server/lib/users/user-api.ts

```diff
@@ -132,6 +137,12 @@ const matchHistoryRetrievalThrottle = createThrottle('matchhistoryretrieval', {
   window: 60000,
 })
 
+const rankingsHistoryRetrievalThrottle = createThrottle('rankingshistoryretrieval', {
+  rate: 50,
```

Comment:
I don't think 50 requests per minute is at all a reasonable rate for this API, this is very much in "someone is scraping the entire userbase" territory, not at all what the general usage would be for real users